src/drivers/github_test.go

```diff
@@ -35,8 +35,7 @@ func TestGitHubDriver_CanMergePullRequest(t *testing.T) {
 	defer teardown()
 	httpmock.RegisterResponder("GET", currentPullRequestURL, httpmock.NewStringResponder(200, `[{"number": 1, "title": "my title" }]`))
 	canMerge, defaultCommintMessage, pullRequestNumber, err := driver.CanMergePullRequest("feature", "main")
-
-	assert.Nil(t, err)
+	assert.NoError(t, err)
```

Comment:
Thanks for removing the empty lines inside function bodies. They should all go away.

Reply:
I could do a quick fixup on all places wothin this very pr.

Reply:
Better to do it as a separate PR. Each PR should "do" only one thing. Otherwise your feature branches live too long and make too many changes, which leads to merge conflicts and long review times. Thanks!